### PR TITLE
copy same format button

### PR DESCRIPTION
**Pull request by junieLer**
add google button copied same format

### CODE REVIEW

File: client/components/Login.jsx [hunk 1]
```diff
@@ -15,12 +22,14 @@ import {
 
 function Login() {
   const [username, setUsername] = useState('');
+>>>>>>> 6b2ddad6dfcd8ea2a90f66ecdeacb9640adf3e5d
```
Comment (AdamrG1) [inline client/components/Login.jsx]: what is the purpose of this?

File: client/components/Login.jsx [hunk 2]
```diff
@@ -99,13 +108,25 @@ function Login() {
...
+            </button>
+          </div>
+          <div id='customBtn' class='customGPlusSignIn'>
+            <span class='icon'></span>
+            <span class='buttonText'>Google</span>
+          </div>
+=======
           <div> </div>
           <button className='buttons' id='signup' onClick={onRegisterClick}>
             Create new account
```
Comment (AdamrG1) [inline client/components/Login.jsx]: why are there two 'create new account' buttons?

**Review by AdamrG1 — CHANGES_REQUESTED** [review]
Please fix the duplicate buttons and add comments.

File: client/components/Login.jsx (hunk 1 above)
Comment (michaelecolley) [reply client/components/Login.jsx]:
```suggestion

```

File: client/components/Login.jsx [hunk 3]
```diff
@@ -99,13 +108,25 @@ function Login() {
           <button className='buttons' id='logInButton' onClick={onLoginClick}>
             Sign in
           </button>
+<<<<<<< HEAD
```
Comment (michaelecolley) [inline client/components/Login.jsx]:
```suggestion

```

File: client/components/Login.jsx [hunk 4]
```diff
@@ -99,13 +108,25 @@ function Login() {
...
+          <div>
+            <button className='buttons' id='signup' onClick={onRegisterClick}>
+              Create new account
+            </button>
+          </div>
+          <div id='customBtn' class='customGPlusSignIn'>
+            <span class='icon'></span>
+            <span class='buttonText'>Google</span>
+          </div>
+=======
```
Comment (michaelecolley) [inline client/components/Login.jsx]:
```suggestion

```

File: client/components/Login.jsx [hunk 5]
```diff
@@ -99,13 +108,25 @@ function Login() {
...
           <div> </div>
           <button className='buttons' id='signup' onClick={onRegisterClick}>
             Create new account
           </button>
           <button className='buttons' id='googleSignup' onClick={googleClick}>
             Create new account with Google
           </button>
+>>>>>>> 6b2ddad6dfcd8ea2a90f66ecdeacb9640adf3e5d
```
Comment (michaelecolley) [inline client/components/Login.jsx]:
```suggestion
```

File: client/components/Login.jsx [hunk 6]
```diff
@@ -1,5 +1,12 @@
 const regeneratorRuntime = require('regenerator-runtime');
 const axios = require('axios');
+<<<<<<< HEAD
+import React, { useState, useEffect } from 'react';
+import { Redirect } from 'react-router-dom';
+
+function Login() {
+  const [username, setUsername] = useState(''); //setUsername to change state - useState assign initial value for state
+=======
```
Comment (michaelecolley) [inline client/components/Login.jsx]:
```suggestion
```

**Review by ronke11 — COMMENTED** [review]
Make changes per previous reviewers

**Pull request closed without merging**